src/config/dynamic-theme-fixes.config

```diff
@@ -11037,6 +11037,21 @@ INVERT
 
 ================================
 
+rain.thecomicseries.com
+
+CSS
+html {
+    background-color: revert !important;
+}
+body {
+    background-color: #333333 !important;
```

Comment:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

Reply:
This actually goes in the wrong direction. As I explained, the issue is that the background color should match the background image, and the only real way to do this is to fix the color to an exact color value. Below is a composite screenshot of what is happening. Left is what Dark Reader does by default, middle is my proposed change that is identical to the original site (which uses a light mode `#container` with a dark background behind it), and left is your proposed change. The screenshots were taken using the dark mode setting, and the issue is reversed when using the light mode setting in Dark Reader.

![darkrain](https://user-images.githubusercontent.com/94094391/141381000-9d87411c-45c7-4474-bce5-583051b87c71.png)

Reply:
> This actually goes in the wrong direction. As I explained, the issue is that the background color should match the background image, and the only real way to do this is to fix the color to an exact color value. Below is a composite screenshot of what is happening. Left is what Dark Reader does by default, middle is my proposed change that is identical to the original site (which uses a light mode `#container` with a dark background behind it), and left is your proposed change. The screenshots were taken using the dark mode setting, and the issue is reversed when using the light mode setting in Dark Reader.

Just remove the background-image?